SU-SWS/saa_victoria_subtheme
# ADAPT-476: Fixed special case of stronger button hover color style.

# READY FOR REVIEW

# Summary
- Fix form button hover

# Review By (Date)
- 7/21

# Urgency
- high

# Steps to Test

1. pull down branch
1. clear site cache
2. go to site/user/login and check the hover state of the sign in button there. it should be dark purple, not red.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
